### PR TITLE
[BUG] Update tests.yml with ACCOUNT_CONSISTENCY #6488

### PR DESCRIPTION
**Pull request by David-Noble-at-work**
See #6488.

### CODE REVIEW

File: sdk/cosmos/tests.yml [hunk 1]
```diff
@@ -1,339 +1,340 @@
...
+                OSVmImage: 'windows-2019'
+                ProfileFlag: '-Pe2e'
+                AdditionalArgs: '-DargLine="-Dazure.cosmos.directModeProtocol=Https"'
+                DESIRED_CONSISTENCY: 'Strong'
+            # 08m 50s, expect passed
+            Strong_Https_Fast:
+                DisplayName: Strong HTTP Fast
+                OSVmImage: 'macOS-10.13'
+                ProfileFlag: '-Pfast'
+                AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
```
Comment (moderakh) [inline sdk/cosmos/tests.yml]: I saw you renamed `ACCOUNT_DEFAULT_CONSISTENCY` to `ACCOUNT_CONSISTENCY` above, which was the right thing to do. seems this one is missed.

As per the code in `/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java` 
The config name is `ACCOUNT_CONSISTENCY` not `ACCOUNT_DEFAULT_CONSISTENCY`


```
    public static String CONSISTENCY =
            System.getProperty("ACCOUNT_CONSISTENCY",
                               StringUtils.defaultString(Strings.emptyToNull(
                                       System.getenv().get("ACCOUNT_CONSISTENCY")), "Strong"));
```

File: sdk/cosmos/tests.yml [hunk 2]
```diff
@@ -1,339 +1,340 @@
...
+                PROTOCOLS: '["Https"]'
+                DESIRED_CONSISTENCIES: '["Strong", "Session"]'
+            # 08m 50s, expect passed
+            Strong_Https_Direct:
+                DisplayName: Strong HTTP Direct
+                OSVmImage: 'macOS-10.13'
+                ProfileFlag: '-Pdirect'
+                AdditionalArgs: '-DargLine="-DACCOUNT_DEFAULT_CONSISTENCY=Strong"'
```
Comment (moderakh) [inline sdk/cosmos/tests.yml]: DACCOUNT_DEFAULT_CONSISTENCY -> ACCOUNT_CONSISTENCY here and other places too.

Comment (David-Noble-at-work) [reply sdk/cosmos/tests.yml]: Thanks for finding this.

**Pull request merged**